Utilities/Conclusion/conclusion.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Documentation/documentation.css">
+    <link rel="stylesheet" href="/stylesheets/utility.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+        <link rel="stylesheet" href="/components/Button/button.css"> 
+        <link rel="stylesheet" href="/Utilities/Conclusion/conclusion.css">
+    <title>Conclusion</title>
+</head>
+<body>
+    <nav class="navigation">
+        <h1>SUPER UI</h1>
+        <ul class="navigation__list">
+          <li><a class="navigation__list--item" href="/index.html">Home</a></li>
+          <li><a class="navigation__list--item nav--active" href="/Documentation/documentation.html">Docs</a></li>
+          <li><a class="navigation__list--item" href="#">Dark</a></li>
+        </ul>
+      </nav>
+      <div class="content">
+  
+        <div class="sidebar">
+  
+          <h3>Get Started</h3>
+          <div class="divider"></div>
+          <ul>
+            <li><a href="/Documentation/documentation.html">Introduction</a></li>
+            <li><a href="/Utilities/Colors/color.html">Colors</a></li>
+            <li><a href="/Utilities/Text Utility/textUtil.html">Text Utility </a></li>
+            <li><a href="/Utilities/Conclusion/conclusion.html">Conclusion</a></li>
+            
+          
+          
+        </ul>
+          <h3>Components</h3>
+          <div class="divider"></div>
```

Comment:
you could have used <hr> tag here, right?

Reply:
Ganesh Ji which tag are you saying I am not able to understand . Can You be more specific?